### PR TITLE
Support for Rollup v1.0.0

### PR DESCRIPTION
**Pull request by panoply**
This PR updates plugin dependencies and now applies support for Rollup v1.0.0 regarding issue #14. This PR also includes dependency updates within the examples directory which now uses Gulp v4.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,27 +121,25 @@ You can also pass a function that returns rollup options object as an argument.
 
 #### `rollup`
 
-You can specify the 3rd argument for replacing `rollup` object by your dependency. It is useful if you want to use a new version of rollup than gulp-rollup-each is using.
+You can specify the 1st argument for replacing `rollup` object by your dependency. It is useful if you want to use a new version of rollup than gulp-rollup-each is using.
```
Comment (ko-yelie) [inline README.md]: Is `3rd` change necessary?

File: README.md [hunk 2]
```diff
@@ -112,27 +121,25 @@ You can also pass a function that returns rollup options object as an argument.
...
-    .pipe(
-      rollupEach(
-        {},
-        {
-          output: {
-            format: 'iife'
-          }
-        },
 
-        // Passing rollup object
```
Comment (ko-yelie) [inline README.md:130]: Please leave this comment.

File: lib/index.js [hunk 3]
```diff
@@ -32,10 +33,12 @@ module.exports = function (arg1, arg2, injectedRollup) {
       arg2 = arg2 == null ? inputOptions.output : arg2
       const outputOptions = typeof arg2 === 'function' ? arg2(file) : arg2
 
+
```
Comment (ko-yelie) [inline lib/index.js]: Don't use multiple empty lines.

File: lib/index.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ const PLUGIN_NAME = 'gulp-rollup-each'
 const cache = {} // cache - (outside of export)
 
 module.exports = function (arg1, arg2, injectedRollup) {
+
```
Comment (ko-yelie) [inline lib/index.js]: Is it your style to put blank lines on the first line?
I would like to leave it as it is without putting empty lines if possible.
I often use JavaScript Standard Style.

**Review by ko-yelie — CHANGES_REQUESTED** [review]

File: lib/index.js (hunk 4 above)
Comment (panoply) [reply lib/index.js]: Hey @ko-yelie –  I generally have ESLint and prettier handling formatting. Maybe worth adding in your style preferences with a `.rc` file.

Comment (ko-yelie) [reply lib/index.js]: I see. 😄 
I will add the `extends` to `.eslintrc`.

**Review by ko-yelie — APPROVED** [review]
Thank you so much!

File: lib/index.js (hunk 4 above)
Comment (panoply) [reply lib/index.js]: Great! Always enjoy working on this poetic little plugin. Happy New Year friend.

**Comment by jonnitto** [comment]
Works like a charm. Thank you a lot!

**Pull request merged**